agent/agent-tooling/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/localstorage/LocalFileLoader.java

```diff
@@ -158,15 +153,15 @@ private static void readFully(FileInputStream fileInputStream, byte[] byteArray,
 
   // either delete it permanently on success or add it back to cache to be processed again later on
   // failure
-  public void updateProcessedFileStatus(boolean successOrNonRetryableError, File file) {
+  void updateProcessedFileStatus(boolean successOrNonRetryableError, File file) {
     if (!file.exists()) {
       // not sure why this would happen
       updateOperationLogger.recordFailure("File no longer exists: " + file.getName());
       return;
     }
     if (successOrNonRetryableError) {
       // delete a file on the queue permanently when http response returns success.
-      if (!LocalStorageUtils.deleteFileWithRetries(file)) {
+      if (!FileUtil.deleteFileWithRetries(file)) {
         // TODO (heya) track file deletion failure via Statsbeat
         updateOperationLogger.recordFailure("Fail to delete " + file.getName());
       } else {
```

Comment:
I know this is not part of this change...I think we discussed and decided that we don't want stats for the file delete failure scenarios.  Can we remove the todo?